docs/source/tutorial/understanding_evaluation.rst

```diff
@@ -24,14 +24,104 @@ triple :math:`(h, r, t) \in \mathcal{T}_{eval}` in this set, two tasks are solve
     relation, and then performing a matrix multiplication with the matrix of all entity embeddings.
     # TODO: Link to section explaining this concept.
 
-Rank-Based Evaluation
----------------------
 In the rank-based evaluation protocol, the scores are used to sort the list of possible choices by decreasing score,
 and determine the *rank* of the true choice, i.e. the index in the sorted list. Smaller ranks indicate better
 performance. Based on these individual ranks, which are obtained for each evaluation triple and each side of the
 prediction (left/right), there exist several aggregation measures to quantify the performance of a model in a single
 number.
 
+.. note::
+
+    There are theoretical implications based on whether the indexing is
+    `0-based <https://en.wikipedia.org/wiki/Zero-based_numbering>`_  or
+    1-based (natural). PyKEEN uses 1-based indexing to conform with related work.
+
+Rank-Based Metrics
+~~~~~~~~~~~~~~~~~~
+Given the set of individual rank scores $\mathcal{I}$, the following scores are commonly used as aggregation.
+
+Hits @ K
+********
+The hits @ k describes the fraction of true entities that appear in the first $k$ entities of the sorted rank list.
+It is given as:
+
+.. math::
+
+    \text{score}_k = \frac{1}{|\mathcal{I}|} \sum \limits_{r \in \mathcal{I}} \mathbb{I}[r \leq k]
+
+For example, if Google shows 20 results on the first page, then the percentage of results that are relevant is the
+hits @ 20. The hits @ k, regardless of $k$, lies on the $[0, 1]$ where closer to 1 is better.
+
+.. warning::
+
+    This metric does not differentiate between cases when the rank is larger than $k$.
+    This means that a miss with rank $k+1$ and $k+d$ where $d \gg 1$ have the same
+    effect on the final score. Therefore, it is less suitable for the comparison of different
+    models.
+
+Mean Rank
+*********
+The mean rank (MR) computes the arithmetic mean over all individual ranks. It is given as:
+
+.. math::
+
+    \text{score} =\frac{1}{|\mathcal{I}|} \sum \limits_{r \in \mathcal{I}} r
+
+It has the advantage over hits @ k that it is sensitive to any model performance changes, not only what occurs
+under a certain cutoff and therefore reflects average performance. With PyKEEN's standard 1-based indexing,
```

Comment:
@cthoyt do you plan to make the indexing-type a parameter in the future? Otherwise, it might be a bit confusing to write what would be, if we had an other indexing scheme.

Reply:
No, I don't think this is something we should implement